docs/windows.rst

```diff
@@ -60,29 +101,33 @@ If you want to create PDB files for debugging even for a release version, you mu
 
         nmake -f mk_mvc.mak PDB=1
 
-GCC
-.............................................................................
+Building ctags with MSBuild
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**General**
+This supports only Microsoft Visual Studio 2013.
```

Comment:
I've just tried with Microsoft Visual Studio Express 2017 and it seems that the project files for 2013 still works. (I haven't tried with 2019 or 2022 yet.)

I had to copy some files before running the `msbuild` command.

```
copy win32\config_mvc.h config.h
copy win32\gnulib_h\langinfo.h gnulib
copy win32\gnulib_h\fnmatch.h gnulib
```

Reply:
Using VS2019 reported following:

```
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(439,5): error MSB8020: The build too 
ls for Visual Studio 2013 (Platform Toolset = 'v120') cannot be found. To build using the v120 build tools, please install Visual Studio 2013 build tools. 
  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarge 
t solution". [C:\Users\vagrant\ctags\win32\ctags_vs2013.vcxproj]
```

I think the file name `ctags_vs2013.*` should rename to other name if it really support VS2015 and VS2017.

Reply:
Ah, this PC has both VS2013 and VS2017 installed. So, even VS2017's msbuild is used, toolset from VS2013 is used.

If `<PlatformToolset>v120</PlatformToolset>` lines in the .vcxproj file are changed to `<PlatformToolset>v141</PlatformToolset>`, it might work on the newer VS versions.

Reply:
As per https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset

```
Visual Studio 2010: v100
Visual Studio 2013: v120
Visual Studio 2015: v140
Visual Studio 2017: v141
Visual Studio 2019: v142
Visual Studio 2022: v143
```

PlatformToolset can't be changed via msbuild options.

It seems that we need `ctags_vs2013.vcxproj` `ctags_vs2015.vcxproj` `ctags_vs2017.vcxproj` `ctags_vs2019.vcxproj` `ctags_vs2022.vcxproj`.

Reply:
It seems that there is a way to support multiple versions of VS in one vcxporj file:
https://qiita.com/yumetodo/items/a8324efaf83c9c08d168 (Japanese)
The basic idea is to switch `PlatformToolset` based on `VisualStudioVersion`.
(This page suggests using CMake instead of this method, though.)

Reply:
Yes, I think we should use `cmake` instead of `msbuild`